### ISSUE TITLE
cli v6.1.1 (nearly) silent on runtime failure

### ISSUE THREAD

**Issue by sei-shissam** [phylum-dev/cli#1365]
# Overview
v6.1.1 running on a linux kernel (< v5.12) which is out-of-date fails with ambiguous error.

# How To Reproduce
Steps to reproduce this behavior:
1. From bash command line run ```uname -a``` and ```phylum --version```
2. Confirm kernel is less than v5.12 and phylum cli is v6.1.1
3. Attempt to run ```phylum analyze  --project myproject-latest go.sum```
4. See ```Error: input/output error: Function not implemented (os error 38)```

# Expected Behavior
A more precise explanation of the error as seen by the phylum cli when run in ```debug``` being ``` Error: unsupported operation, please ensure Kernel version is at least 5.12: Function not implemented (os error 38)```

# Additional Context
cli v6.1.1 silent on runtime failure

running ```phylum analyze``` after ```phylum update```  I got (Function not implemented)

```
$ phylum analyze --project myproject-latest go.sum
❗ Error: input/output error: Function not implemented (os error 38)

❗ Error: Could not parse dependency file "go.sum" as "auto" type

Caused by:
    Dependency file parsing failed
```

Downgrading to ```v5.9.0``` does not exhibit this error.

Cloning/building ```v6.1.1``` and running the same command reports:

```
~/src/phylum-cli/target/debug/phylum analyze --project myproject-latest-gosum go.sum
❗ Error: unsupported operation, please ensure Kernel version is at least 5.12: Function not implemented (os error 38)

❗ Error: Could not parse dependency file "go.sum" as "auto" type

Caused by:
```

Now I know the problem and what to do - it would be great if the ```release``` version of the cli would have this additional informaiton.

**Comment by kylewillmon** [comment]
Thanks for the report.

The error message was improved in 9fdb5e9, which was not included in the v6.1.1 release. We will be releasing v6.1.2 shortly with this fix.

**Comment by maxrake** [comment]
This issue was closed due to the linked PR closing...but the release is still building and should be available in about an hour or less.
